desk/src/components/List/ListFilters.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+	<NestedPopover>
+		<template #target>
+			<Button label="Filter">
+				<template #prefix>
+					<FeatherIcon name="filter" class="h-3.5" />
+				</template>
+				<template v-if="filters?.length" #suffix>
+					<div
+						class="flex h-5 w-5 items-center justify-center rounded bg-gray-900 pt-[1px] text-2xs font-medium text-white"
+					>
+						{{ filters?.length }}
+					</div>
+				</template>
+			</Button>
+		</template>
+
+		<template #body="{ close }">
+			<div class="rounded-lg border border-gray-100 bg-white p-4 shadow-xl">
+				<div
+					v-if="filters?.length"
+					v-for="({ fieldname, fieldtype, operator, value, options }, i) in filters"
+					class="mb-3 flex items-center gap-2"
+				>
+					<div class="w-[9rem]">
+						<Autocomplete
+							:body-classes="'w-[15rem]'"
```

Comment:
does this affect it since the outer width is already set? Also maybe avoid arbitrary values for tailwind styles?